wgengine/router/runner.go

```diff
@@ -55,7 +56,11 @@ func (o osCommandRunner) output(args ...string) ([]byte, error) {
 		return nil, errors.New("cmd: no argv[0]")
 	}
 
-	out, err := exec.Command(args[0], args[1:]...).CombinedOutput()
+	cmd := exec.Command(args[0], args[1:]...)
+	cmd.SysProcAttr = &syscall.SysProcAttr{
+		AmbientCaps: []uintptr{uintptr(12)},
```

Comment:
12 looks super weird.

use a named constant. one surely exists already.

also, big fat comment what's going on here and why. should we only do this on Synology? any harm in doing it on other Linux?

Reply:
looks like `unix.CAP_NET_ADMIN` ?

Reply:
Not sure if there is any harm to do it on other distros, it's safer to do Synology only